superset/sql_lab.py

```diff
@@ -14,14 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=C,R,W
 import logging
 import uuid
 from contextlib import closing
 from datetime import datetime
 from sys import getsizeof
 from typing import Optional, Tuple, Union
 
+# pylint and isort disagree on the correct import order.
+# Let's have isort win.
+# pylint: disable=ungrouped-imports
```

Comment:
should we add this to a base pylint rule disable? so that we don't need it on a bunch of files?

Reply:
I’m a little perflexed as to why `pylint` is complaining here.

Reply:
Yup. Not a bad idea. Will update here before I forget.